### PR TITLE
Turkish Translation Update

### PR DESCRIPTION
**Pull request by umitseyhan75**
It's done, hope it will help you.

### CODE REVIEW

File: localization/languages/tr.json [hunk 1]
```diff
@@ -138,37 +138,37 @@
...
+        "settingsDarkModeNever": "Asla",
+        "settingsDarkModeNight": "Geceleyin",
+        "settingsDarkModeAlways": "Her zaman",
         "settingsSiteThemeToggle": "Site temasını etkinleştir",
         "settingsAdditionalFeaturesHeading": "Ek Özellikler",
         "settingsUserscriptsToggle": "Kullanıcı betiklerini etkinleştir",
-        "settingsSeparateTitlebarToggle": null, //missing translation
-        "settingsOpenTabsInForegroundToggle": null, //missing translation
+        "settingsSeparateTitlebarToggle": "Ayrılmış başlık çubuğu kullan"
+        "settingsOpenTabsInForegroundToggle": "Bir bağlantıyı yeni sekmede açtığımda hemen o sekmeye geç"
```
Comment (PalmerAL) [inline localization/languages/tr.json:153]: This is the wording used in Firefox, right?

Min uses a different description than Firefox for this, so can you make the translation match the English setting name in Min?

File: localization/languages/tr.json [hunk 2]
```diff
@@ -220,20 +220,50 @@
...
-        "autoRedirectBannerReader": null, //missing translation "Always open articles from this site in reader view?"
-        "buttonReaderRedirectYes": null, //missing translation "Yes"
-        "buttonReaderRedirectNo": null, //missing translation "No"
-        "articleReaderView": null, //missing translation "Original Article"
+        "buttonReaderSettings": "Okuyucu Ayarları",
+        "buttonReaderLightTheme": "Açık",
+        "buttonReaderSepiaTheme": "Sepya",
+        "buttonReaderDarkTheme": "Koyu",
+        "openReaderView": "Her zaman okuyucu görünümünde aç",
+        "autoRedirectBannerReader": "Bu sitedeki makaleler her zaman okuyucu görünümünde açılsın mı?,
```
Comment (PalmerAL) [inline localization/languages/tr.json:228]:
```suggestion
        "autoRedirectBannerReader": "Bu sitedeki makaleler her zaman okuyucu görünümünde açılsın mı?",
```

**Comment by PalmerAL** [comment]
Everything looks good except for the few things I commented on.

**Comment by umitseyhan75** [comment]
I hope I did'nt make another mistake.

**Comment by PalmerAL** [comment]
Everything looks good, thank you!

**Pull request merged**